Library/Homebrew/keg_relocate.rb

```diff
@@ -87,15 +87,21 @@ def change_rpath(file, new_prefix)
     end
     return unless patchelf.installed?
     cmd = "#{patchelf.bin}/patchelf --set-rpath #{new_prefix}/lib"
+    test_cmd = "#{patchelf.bin}/patchelf --print-rpath --print-interpreter #{file}"
+    (old_i, old_p) = `#{test_cmd}`.split("\n")
```

Comment:
Wanted to keep below 80ish characters per line, but if you think this is justified I'll go with it.